### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by galkin**
@hackygolucky, please look this pull.

**Before**:
Before

**After**:
After

### CODE REVIEW

File: certification.md [hunk 1]
```diff
@@ -129,21 +129,14 @@ Recertification requirements and interval:
...
-
-1. No surveillance activities are planned to evaluate certificants on an ongoing
-basis.
+3. The exam will reviewed and updated every 2 years.
+4. The LTS release support cycle reflects the nature and maturity of the industry or field in which the certified person is working.
+5. The risks resulting from an incompetent person include the following:  bad hire; lost recruitment money; incremental training costs; missed deadlines.
+6. Ongoing changes in technology, and requirements for certified persons are reflected in ES releases.
+7. Requirements of interested parties were taken into account in formulating the task force, and will be included in guiding bodies for updating and revision of the exam.  Interested parties include:
+    - Programmers  Training companies  Foundation  Consultancy companies
+    - Organizations using Node  Hiring managers
```
Comment (addaleax) [inline certification.md]: I think these are supposed to be lists, too; the rendered document seems to say “Organizations which use hiring managers for Node” but really “Organizations using Node” and “Hiring managers” are supposed to be distinct items (again, that’s what I think :smile:)

Comment (galkin) [reply certification.md]: @addaleax fixed

**Review by addaleax — APPROVED** [review]

**Comment by rsp** [comment]
@galk-in @addaleax This is related to my PR #19 that was merged into [certification-domains-doc](https://github.com/nodejs/education/tree/certification-domains-doc) branch by @hackygolucky.

**Comment by galk-in** [comment]
@hackygolucky @addaleax do you have any updates?

**Pull request closed without merging**